src/code_102610_len_2330.c

```diff
@@ -30,13 +30,13 @@ void func_802E10F4(Entity* entity) {
 }
 
 void func_802E114C(void) {
-    func_80027088(TRUE);
+    func_80027088(1);
```

Comment:
why

Reply:
This function takes 2 sometimes, so it's definitely not just a bool